### PR TITLE
MODORDERS-775 - Add deduplication mechanism for Create Order event

### PR DESCRIPTION
**Pull request by RuslanLavrov**
## Purpose
to avoid events duplication during data import for order creation

## Approach
* add orderIdStorage implementation to handle possible events duplication
* add tests
* fix tests run on jenkins

## Learning
[MODORDERS-775](https://issues.folio.org/browse/MODORDERS-775)

### CODE REVIEW

**Review by KaterynaSenchenko — APPROVED** [review]

File: src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "tables": [],
+  "scripts": [
+    {
+      "run": "after",
+      "snippetPath": "create_records_orders_table.sql",
+      "fromModuleVersion": "mod-source-record-manager-12.6.0"
```
Comment (okolawole-ebsco) [inline src/main/resources/templates/db_scripts/schema.json:7]: should this reference mod-orders instead?

File: src/main/resources/templates/db_scripts/create_records_orders_table.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-- Creates table to store association between record id and order id
+CREATE TABLE IF NOT EXISTS records_orders (
```
Comment (okolawole-ebsco) [inline src/main/resources/templates/db_scripts/create_records_orders_table.sql:2]: is record here a Kafka record?

does this table get cleaned out sometimes?

**Review by SerhiiNosko — APPROVED** [review]

File: src/main/resources/templates/db_scripts/schema.json (hunk 1 above)
Comment (RuslanLavrov) [reply src/main/resources/templates/db_scripts/schema.json]: sure, thanks!

File: src/main/resources/templates/db_scripts/create_records_orders_table.sql (hunk 2 above)
Comment (RuslanLavrov) [reply src/main/resources/templates/db_scripts/create_records_orders_table.sql:2]: here it means an srs record. For now, this table is not cleaned.

**Review by Aliaksandr-Fedasiuk — APPROVED** [review]

**Review by VRohach — APPROVED** [review]
@RuslanLavrov Perfect work!

**Review by okolawole-ebsco — APPROVED** [review]

**Pull request merged**